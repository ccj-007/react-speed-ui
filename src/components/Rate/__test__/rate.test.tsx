import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import { Rate } from '../../index';

describe('test Rate component', () => {
  it('should render', () => {
    const { asFragment } = render(<Rate />);
    expect(asFragment()).toMatchSnapshot();
  });
});
