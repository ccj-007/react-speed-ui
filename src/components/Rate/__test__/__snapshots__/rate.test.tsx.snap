// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Rate component should render 1`] = `
<DocumentFragment>
  <div
    class="speed-rate"
  >
    <div
      class="iconfont speed-rate-item icon-star"
      style="font-size: 32px;"
    />
    <div
      class="iconfont speed-rate-item icon-star"
      style="font-size: 32px;"
    />
    <div
      class="iconfont speed-rate-item icon-star"
      style="font-size: 32px;"
    />
    <div
      class="iconfont speed-rate-item icon-star"
      style="font-size: 32px;"
    />
    <div
      class="iconfont speed-rate-item icon-star"
      style="font-size: 32px;"
    />
    <div
      class="speed-rate-text"
    />
  </div>
</DocumentFragment>
`;
